pkg/balance/balance.go

```diff
@@ -0,0 +1,112 @@
+package balance
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/safex/gosafex/internal/crypto/derivation"
+	"github.com/safex/gosafex/pkg/safex"
+	"github.com/safex/gosafex/pkg/safexdrpc"
+)
+
+// Containing balance status
+type Balance struct {
+	CashUnlocked  uint64
+	CashLocked    uint64
+	TokenUnlocked uint64
+	TokenLocked   uint64
+}
+
+type KeyType = []byte
```

Comment:
In the future, use a single Key type across the project - account.Key or similar.